content/blog/cats-case-study.md

```diff
@@ -0,0 +1,71 @@
++++
+authors = ["Sutirtha Das", "Padma B" ]
+categories = ["Case Study"]
+date = 2023-02-03T00:00:00Z
+description = ""
+draft = false
+image = "/images/2023/finance.jpg"
+slug = "finance-modernization-case-study"
+tags = ["Case Study", "Modernization"]
+title = "A Centralized System for a Leading Consumer Lending Bank "
++++
+
+### Client:
+
+A leading customer credit provider in the United States, committed to assisting customers in achieving their personal objectives through great customer service and flexible lending alternatives.
+
+### Problem statement:
+
+The bank has over 30 million unique loan contracts spread across various departments. As the number of contracts grew over time, handling all these contracts manually led to these major drawbacks.
+
+1. No centralized system to access all the contracts and work on them. The agents had to do multiple logins to different branch systems to work on these contracts and search across various branches even find the correct contract.
```

Comment:
Not only different branches but different systems for different workflows as well. And the same in the solution - all workflows in the same system. 
One system to rule them all.

Reply:
lol. updated slightly